modules/photon-cdn.php

```diff
@@ -108,6 +123,19 @@ public static function get_core_assets( $version = null, $locale = null ) {
 			$locale = get_locale();
 		}
 
+		/**
+		 * Filters Jetpack CDN's Core version number and locale. Can be used to override the values
+		 * that Jetpack uses to retrieve assets. Expects the values to be returned in an array.
+		 *
+		 * @since 6.6
+		 *
+		 * @param array $values array( $version  = core assets version, i.e. 4.9.1, $locale = desired locale )
+		 */
+		list( $version, $locale ) = apply_filters(
+			'jetpack_cdn_plugin_slug_and_version',
```

Comment:
this one runs different parameters than the other, and should probably have a different filter name.  `jetpack_cdn_core_version_and_locale` maybe?

Reply:
Yes, copypasta, good catch, thank you!